md/BonitaBPM_platform_setup.md

```diff
@@ -28,7 +28,8 @@ It is composed of the following items:
     * `current/`: contains configuration files after a `pull` from the database is made.
     * `licenses/`: (Subscriptions only) folder to put the license file to allow Bonita BPM Platform to start without error.
     * `sql/`: SQL scripts used to create Bonita BPM database tables
-* `database.properties`: contains properties to connect to the database on which the configuration is managed.
+* `database.properties`: used as a simplified form to get values from the system administrator. It contains properties needed to connect to the database. 
```

Comment:
used as a simplified entry form to get property values to connect to the database. Those values will be used by the file `internal.properties`.